### PR TITLE
Enable environment new structure

### PR DESCRIPTION
**Pull request by aman19K**
- [x] enable environment new structure
- [x] import minor fix

### CODE REVIEW

File: packages/contentstack-import/src/commands/cm/stacks/import.ts [hunk 1]
```diff
@@ -110,18 +110,17 @@ export default class ImportCommand extends Command {
       // Note setting host to create cma client
       importConfig.host = this.cmaHost;
       contentDir = importConfig.contentDir;
-      backupDir = importConfig.backupDir;
       const managementAPIClient: ContentstackClient = await managementSDKClient(importConfig);
       const moduleImporter = new ModuleImporter(managementAPIClient, importConfig);
       await moduleImporter.start();
       log(importConfig, `The content has been imported to the stack ${importConfig.apiKey} successfully!`, 'success');
-      log(importConfig, `The log has been stored at '${path.join(importConfig.backupDir, 'logs', 'import')}'`, 'success');
+      log(importConfig, `The log has been stored at '${path.join(contentDir, 'logs', 'import')}'`, 'success');
```
Comment (shafeeqd959) [inline packages/contentstack-import/src/commands/cm/stacks/import.ts:117]: Is it content dir/ Back up dir where logs are getting created

File: packages/contentstack-import/src/import/modules/base-class.ts [hunk 2]
```diff
@@ -306,11 +306,11 @@ export default abstract class BaseClass {
       case 'update-gfs':
         return apiData.update().then(onSuccess).catch(onReject);
       case 'create-environments':
-      // return this.stack
-      //   .environment()
-      //   .create({ environment: omit(apiData, ['uid']) as EnvironmentData })
-      //   .then(onSuccess)
-      //   .catch(onReject);
+        return this.stack
```
Comment (shafeeqd959) [inline packages/contentstack-import/src/import/modules/base-class.ts:309]: Can you update the management sdk version wherever required ?

Comment (aman19K) [reply packages/contentstack-import/src/import/modules/base-class.ts:309]: @shafeeqd959 done

File: packages/contentstack-import/src/commands/cm/stacks/import.ts (hunk 1 above)
Comment (aman19K) [reply packages/contentstack-import/src/commands/cm/stacks/import.ts:117]: @shafeeqd959 content directory.

**Review by shafeeqd959 — APPROVED** [review]

**Review by shafeeqd959 — APPROVED** [review]

**Pull request merged**